tests/org.dataflowanalysis.converter.tests/src/org/dataflowanalysis/converter/tests/PCMTest.java

```diff
@@ -33,32 +35,40 @@
 
 import tools.mdsd.library.standalone.initialization.StandaloneInitializationException;
 
-public class PCMTest {
+public class PCMTest extends ConverterTest{
     @Test
-    @DisplayName("Test Palladio to DFD")
-    public void palladioToDfd() {
-        String modelLocation = "org.dataflowanalysis.converter.testmodels";
-
-        testSpecificModel("TravelPlanner", "travelPlanner", modelLocation, null, null);
+    @DisplayName("Test PCM2DFD TravelPlanner")
+    public void travelToDfd() {
+        testSpecificModel("TravelPlanner", "travelPlanner", TEST_MODELS, null, null);
+    }
+	
+	@Test
+    @DisplayName("Test PCM2DFD MaaS")
+    public void maasToDfd() {
+        testSpecificModel("MaaS_Ticket_System_base", "MaaS", TEST_MODELS, "maas.json", null);
+    }
+	
+	@Test
+    @DisplayName("Test PCM2DFD CWA")
+    public void cwaToDfd() {
+        testSpecificModel("CoronaWarnApp", "default", TEST_MODELS, "cwa.json", null);
     }
 
     @Test
     @Disabled("There is currently no manually converted pcm model")
     @DisplayName("Test manual Palladio to DFD")
     public void manualPCMToDfd() throws StandaloneInitializationException {
-        String modelLocation = "org.dataflowanalysis.converter.testmodels";
-
         String inputModel = "InternationalOnlineShop";
         String inputFile = "default";
         String dataflowdiagram = Paths.get("models", "OnlineShopDFD", "onlineshop.dataflowdiagram")
                 .toString();
         String datadictionary = Paths.get("models", "OnlineShopDFD", "onlineshop.datadictionary")
                 .toString();
-        testSpecificModel(inputModel, inputFile, modelLocation, null,
-                new DataFlowDiagramConverter().loadDFD(modelLocation, dataflowdiagram, datadictionary, Activator.class));
+        testSpecificModel(inputModel, inputFile, TEST_MODELS, null,
+                new DataFlowDiagramConverter().loadDFD(TEST_MODELS, dataflowdiagram, datadictionary, Activator.class));
 
     }
-
+    
     private void testSpecificModel(String inputModel, String inputFile, String modelLocation, String webTarget,
```

Comment:
Couldn't you extract `transform(...), checkDFD(...)` and `storeWeb(...)` from that to get rid of the `null`s?
If not, please document that, because currently it is quite hard to read